README.md

```diff
@@ -121,6 +121,12 @@ Realm Flutter package is published to [realm](https://pub.dev/packages/realm).
     ```
     flutter pub add realm
     ```
+* (Optional) Install the `realm` package into the application. This downloads and copies the required native binaries to the app directory. 
```

Comment:
I suggest we make it clear from the beginning what is this about. So something like

`For running flutter widget and unit tests run the following command to install the required native binaries`

Reply:
OK